File: backend/tiggle-root/tiggle/src/main/java/com/side/tiggle/domain/txtag/model/TxTag.java
package com.side.tiggle.domain.txtag.model;

import com.side.tiggle.global.common.model.BaseEntity;
import lombok.*;

import javax.persistence.*;

@Entity
@Data
@EqualsAndHashCode(callSuper = false)
@Builder
@NoArgsConstructor(access = AccessLevel.PROTECTED)
@AllArgsConstructor
@Table(name = "tx_tags")
public class TxTag extends BaseEntity {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(name = "tx_id", nullable = false)
    private Long txId;

    @Column(name = "member_id", nullable = false)
    private Long memberId;

    @Column(name = "tag_names", nullable = false)
    private String tagNames;

    @Builder
    public TxTag(Long txId, Long memberId, String tagNames) {
        this.txId = txId;
        this.memberId = memberId;
        this.tagNames = tagNames;
    }
}

File: backend/tiggle-root/tiggle/src/main/kotlin/com/side/tiggle/domain/txtag/model/TxTag.kt
package com.side.tiggle.domain.txtag.model

import javax.persistence.*

@Entity
@Table(name = "tx_tags")
class TxTag(
    @Column(name = "tx_id", nullable = false)
    val txId: Long,

    @Column(name = "member_id", nullable = false)
    val memberId: Long,

    @Column(name = "tag_names", nullable = false)
    var tagNames: String
) {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Long = 0
}